src/blog/2023/011-form-js-1.6.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Unveiling Dynamic Forms
+description: "Explore what's new in form-js@1.6.0: Unveil the power of dynamic lists and an innovative table component, designed to enhance the functionality and versatility of your forms."
+preview_image: http://bpmn.io/assets/attachments/blog/2023/011-dynamic-table.png
+layout: blogpost.hbs
+slug: 2023-form-js-1-6-0-unveiling-dynamic-forms
+author:
+  - Vinicius Goulart <https://github.com/vsgoulart>
+  - Valentin Serra <https://github.com/skaiir>
+published: 2023-12-18 12:00
+
+releases:
+  - "form-js@1.6.0"
+---
+
+<p class="introduction">
+  We're thrilled to announce the [form-js@1.6.0 release](https://github.com/bpmn-io/form-js), a significant leap in form development. This version introduces dynamic lists and a new table component, elevating the capabilities of complex form creation.
+</p>
+
+<!-- continue -->
+
+Explore these new features [on our demo](https://demo.bpmn.io/form) or using the [Camunda form playground](https://github.com/camunda/form-playground).
+
+## Dynamic Lists
+
+Without manual work, dynamic lists enable the effortless generation of repetitive sets of form components, such as data tables or ordered lists.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-dynamic-list.png">
+
+  <p class="caption">
+    Dynamic lists offer flexibility in modeling repetitive lists of form components.
```

Comment:
Consider: Link this image to your demo (camunda forms).

Better: Get the bpmn.io form demo to feature the form playground, and link that.